### DISCUSSION TITLE
Weird triangles appear when I import template weapon models from Arma 3 Samples

### DISCUSSION THREAD

**Discussion by Tiggy1473** [AlwarrenSidh/ArmAToolbox#61] [Q&A]
Currently using Blender 4.1. installed armatools by downloading the release and selecting the zip archive with blender.
![image](https://github.com/AlwarrenSidh/ArmAToolbox/assets/55516215/79d1f3c0-7ab2-4419-83b0-a74c355a6ad0)

**Comment by AlwarrenSidh** [comment]
These triangles are proxies, and they are supposed to be there.
